docs/dev/table/connectors/formats/avro-confluent.md

```diff
@@ -45,29 +45,132 @@ Dependencies
     connector=connector
 %}
 
-How to create a table with Avro-Confluent format
-----------------
-
-Here is an example to create a table using Kafka connector and Confluent Avro format.
+Usage examples
+--------------
 
 <div class="codetabs" markdown="1">
 <div data-lang="SQL" markdown="1">
+
+Example of a table using raw UTF-8 string as Kafka key and Avro records registered in the Schema Registry as Kafka values:
+
 {% highlight sql %}
-CREATE TABLE user_behavior (
-  user_id BIGINT,
-  item_id BIGINT,
-  category_id BIGINT,
-  behavior STRING,
-  ts TIMESTAMP(3)
+CREATE TABLE user_created (
+
+  -- one column mapped to the Kafka raw UTF-8 key
+  the_kafka_key STRING,
+  
+  -- a few columns mapped to the Avro fields of the Kafka value
+  id STRING,
+  name STRING, 
+  email STRING
+
 ) WITH (
+
   'connector' = 'kafka',
+  'topic' = 'user_events_example1',
   'properties.bootstrap.servers' = 'localhost:9092',
-  'topic' = 'user_behavior',
-  'format' = 'avro-confluent',
-  'avro-confluent.schema-registry.url' = 'http://localhost:8081',
-  'avro-confluent.schema-registry.subject' = 'user_behavior'
+
+  -- UTF-8 string as Kafka keys, using the 'the_kafka_key' table column
+  'key.format' = 'raw',
+  'key.fields' = 'the_kafka_key',
+
+  'value.format' = 'avro-confluent',
+  'value.avro-confluent.schema-registry.url' = 'http://localhost:8082',
+  'value.fields-include' = 'EXCEPT_KEY'
 )
 {% endhighlight %}
+
+To which  we can write as follows):
```

Comment:
I don't fully understand this sentence. Do you mean "We can write data into the kafka table as follows" ?

Reply:
Thanks for the suggestion. Yes, indeed, that's what I meant. 
I'll add a commit to use your version, it's clearer.